### PR TITLE
Add scaffolder http actions

### PR DESCRIPTION
**Pull request by nicolasarnold12321**
Signed-off-by: Nicolas Arnold <nic@roadie.io>

### CODE REVIEW

File: plugins/scaffolder-backend-module-http-request/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const path = require('path');
```
Comment (punkle) [inline plugins/scaffolder-backend-module-http-request/.eslintrc.js:1]: Should we include this task in the backend?

Comment (punkle) [inline plugins/scaffolder-backend-module-http-request/.eslintrc.js:1]: Should we include this task in the backend?

Comment (nicolasarnold12321) [reply plugins/scaffolder-backend-module-http-request/.eslintrc.js:1]: I think we should!

Comment (nicolasarnold12321) [reply plugins/scaffolder-backend-module-http-request/.eslintrc.js:1]: It's currently using it from source

Comment (punkle) [reply plugins/scaffolder-backend-module-http-request/.eslintrc.js:1]: What do you mean it is using it from source?

File: plugins/scaffolder-backend-module-http-request/.github/ISSUE_TEMPLATE/bug_template.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (punkle) [inline plugins/scaffolder-backend-module-http-request/.github/ISSUE_TEMPLATE/bug_template.md:1]: Is this needed?

Comment (nicolasarnold12321) [reply plugins/scaffolder-backend-module-http-request/.github/ISSUE_TEMPLATE/bug_template.md:1]: Maybe not but it was included in a few other plugins.

**Pull request merged**